data-science-pipelines-operator/base/params.env

```diff
@@ -0,0 +1,9 @@
+IMAGES_APISERVER=registry.redhat.io/rhods/odh-ml-pipelines-api-server-rhel8:latest
+IMAGES_ARTIFACT=registry.redhat.io/rhods/odh-ml-pipelines-artifact-manager-rhel8:latest
+IMAGES_PERSISTENTAGENT=registry.redhat.io/rhods/odh-ml-pipelines-persistenceagent-rhel8:latest
+IMAGES_SCHEDULEDWORKFLOW=registry.redhat.io/rhods/odh-ml-pipelines-scheduledworkflow-rhel8:latest
+IMAGES_VIEWERCRD=registry.redhat.io/rhods/odh-ml-pipelines-viewercontroller-rhel8:latest
+IMAGES_CACHE=registry.redhat.io/rhods/odh-ml-pipelines-cache-rhel8:latest
+IMAGES_MOVERESULTSIMAGE=registry.redhat.io/ubi8/ubi-micro:8.7
+IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
+IMAGES_DSPO=quay.io/opendatahub/data-science-pipelines-operator:latest
```

Comment:
@zdtsw & @jgarciao -- I've added the most recent changes from upstream which parameterize the dspo image (so it is no longer in kustomization.yaml). 


also, from what I understand after talking with @gmfrasca, we do not get the registry.redhat.io image for dspo until the 1.24 build which has not yet happened. 

See also the updated the PR here: 

https://github.com/red-hat-data-services/odh-deployer/pull/308/files 

This shows how these images in params.env (including the dspo deployment image itself) are defaults, and the actual images will be subbed in via env vars via kfdef (you can test by buildling the kfdef ). 

I got these values from [here](https://gitlab.cee.redhat.com/data-hub/rhods-cpaas-midstream/-/blob/rhods-1.25-rhel-8/distgit/containers/odh-operator-bundle/bundle-template/manifests/rhods-operator.clusterserviceversion.yml#L638-651). 

So I'm wondering if this is sufficient to consider this a non-blocker. I.e. keep upstream for dspo here for now, as it will be replaced anyways by the deployer. Once we have 1.24 img in registry.redhat.io, we can update the default as well. WDYT?

Reply:
looks reasonable :+1: , then you no need hardcode which registery to use instead of using env from CSV which suites for different test env. and prod/live

Reply:
@zdtsw for now yes, this is the `default` image, it will be replaced by [this](https://github.com/red-hat-data-services/odh-deployer/pull/308/files#diff-2f0e22459eb141eeef3196c02a3e09b1ce70ed56e170581c24dd1c83cbfd4480R22) image in rhods via odh-deployer. 

> also, from what I understand after talking with @gmfrasca, we do not get the registry.redhat.io image for dspo until the 1.24 build which has not yet happened.

I'm thinking we update the `default` value in this pr, which is just the default, when we have the registry.redhat.io image. WDYT?

Reply:
Sure, as a temp workaround, I am ok with this image. As long as we do not forget to update it once it is available

Reply:
> As long as we do not forget to update it once it is available

@zdtsw I've created this [jira issue](https://issues.redhat.com/browse/RHODS-7708) so that we are sure to get this done as soon as we have the registry.redhat.io image.